Library/Homebrew/cmd/audit.rb

```diff
@@ -234,6 +234,32 @@ def audit_options
     end
   end
 
+  def audit_desc
+    # For now, only check the description when using `--strict`
+    return unless @strict
+
+    desc = formula.desc
+
+    unless desc
+      problem "All formulas should have a desc."
+      return
+    end
+
+    # Make sure the formula name plus description is no longer than 80 characters
+    linelength = formula.name.length + ": ".length + desc.length
+    if linelength > 80
+      problem "Description is too long. \"name: desc\" should be less than 80 characters (currently #{linelength})."
+    end
+
+    if desc =~ %r[[Cc]ommandline]
+      problem "It should be spelled \"command-line\", not \"commandline\"."
```

Comment:
UK tends to favour "spelt" over "spelled". It's a silly thing, but we've had PRs before trying to change single words to various British or American localisations. I'd probably just cut the word entirely to avoid the future debate since the sentence works without it, but if none of the maintainers mind the spelling I'm fine with it.